base/uk.ac.stfc.isis.ibex.epics/src/uk/ac/stfc/isis/ibex/epics/observing/TransformingObservable.java

```diff
@@ -49,28 +48,44 @@ public void onConnectionStatus(boolean isConnected) {
 		}
 
 	};
-
-    protected final void setSource(ClosableObservable<T1> source) {
+    
+    protected void setSource(ClosableObservable<T1> source) {
```

Comment:
Yes but the previous implementation in Transforming Observable wasn't. Typically `setSource` is only called in the constructor so it's unlikely to be an issue. Strictly speaking though it would probably be a good idea so I'll add it back in